munimap/static/sass/components/_geoeditor.sass

```diff
@@ -7,18 +7,9 @@
   &.badge-red
     background-color: rgba(217, 83, 79, 0.8)
 
-.geoeditor
-  .geoeditor-buttons
-    display: table
-
-  .geoeditor-button-wrap
-    display: table-row
-    width: 100%
-
-  .geoeditor-button
-    display: table-cell
-    width: 100%
+$sidebar-width: 300px // TODO: import value
```

Comment:
should this TODO be done before merging?

Reply:
good point. I will see if I can do that.